frame/benchmarking/src/analysis.rs

```diff
@@ -147,6 +169,14 @@ fn linear_regression(
 	mut ys: Vec<f64>,
 	x_vars: usize,
 ) -> Option<(f64, Vec<f64>, Vec<f64>)> {
+	let (intercept, params, errors) = raw_linear_regression(&xs, &ys, x_vars, true)?;
+	if intercept > 0.0 {
+		return Some((intercept, params, errors[1..].to_vec()))
+	}
```

Comment:
This should actually be `>=` right?

Reply:
Hmm.... well, I guess *technically* yes, you're right, although in practice I don't think linear interpolation's going to ever emit an intercept that's actually zero since the numbers would have to perfectly line up for that to happen? At least for the execution times. For other metrics I guess it could maybe happen? I'm not sure.

I'll put up a PR and change it to `>=`.

Reply:
Probably does not matter much for floating point number :man_shrugging: but if you want to fix it, okay